configure

```diff
@@ -5367,7 +5395,13 @@ if test "$_opengl_mode" != none ; then
 						darwin*)
 							_opengl_mode=gl
 							;;
-
+						emscripten)
+							_opengl_mode=gles2
+							_opengl_game_es2=yes
+							_opengl_glad=yes
+							_tinygl=no
```

Comment:
I removed disabling TinyGL by default. I wanted to save the extra binary size since software rendering is quite slow, but I can easily do that by just adding `--disable-tinygl` to the demo build script